### DISCUSSION TITLE
How to integrate a new customized schema(like I want add a customized rgb21 schema)

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#62] [Q&A] [answered]
Hi there, I have some questions here: now if I want to integrate my customized rgb21 schema to a dapp, are there any samples to research, such as if I want to integrate an rgb21 customized schema to the rgb-lightning-node, how can I do it?

**Comment by dr-orlovsky** [comment, marked as answer]
Can you show what the customized schema looks like? Since it is not clear what do you mean by "customized": there is no "standard/non-customized" schema.
